static/sass/styles.scss

```
@@ -297,6 +297,13 @@ body {
 }
 
 .p-details-tab__content {
+  .p-side-navigation {
+    @media (min-width: $breakpoint-medium) {
+      position: sticky;
+      top: 56px;
+    }
```

You'd still need to set `top: 56px`, but the side navigation pattern has an `is-sticky` class you could use: https://vanillaframework.io/docs/patterns/navigation#sticky
```suggestion
    top: 56px;
```